integration/commands_administration_test.go

```diff
@@ -181,8 +181,7 @@ func TestCommandsAdministrationListDatabases(t *testing.T) {
 		},
 		"Regex": {
 			filter: bson.D{
-				{Key: "name", Value: name},
-				{Key: "name", Value: primitive.Regex{Pattern: "^Test", Options: "i"}},
+				{Key: "name", Value: primitive.Regex{Pattern: "ListDatabases$", Options: "i"}},
```

Comment:
This test creates `TestCommandsAdministrationListDatabases_extra` database too. Updated regular expression to return expected database name.